app/battle/battle.py

```diff
@@ -0,0 +1,29 @@
+from app.knights.effects import apply_effects
+from app.knights.armor_and_weapon import apply_armor_and_weapon
+from typing import Dict
+
+
+def battle(knightsconfig: Dict[str, Dict]) -> Dict[str, int]:
+
+    for knight in knightsconfig.values():
+        apply_armor_and_weapon(knight)
+        apply_effects(knight)
+
+    # Fight sequences

```

Comment:
There is no need for comments, remove them since code is self-descriptive enough